vm-setup/scripts/start.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+# Note: Windows Bash doesn't support shebang extra params
+set -e
+
+# Network settings
+sysctl -w net.ipv4.ip_local_port_range="1024 65535"
+sysctl -w net.ipv4.tcp_max_syn_backlog=4096
+sysctl -w net.core.somaxconn=1024
+
+cd /app
+
+source "env-vars"
+
+COL_NO="\033[0m" # no color
+COL_WARN="\033[1;33m" # light yellow
+COL_ERR="\033[1;31m" # light red
+
+APP_PATH="/app"
+WEBUICONFIG="${APP_PATH}/webui-config.js"
+WEBUICONFIG_SAFE="${APP_PATH}/webui-config.js.safe"
+WEBUICONFIG_UNSAFE="${APP_PATH}/webui-config.js.unsafe"
```

Comment:
What is the difference between safe and unsafe modes?  Is there a third mode here as well, and if so, how is it different from the other two?

Reply:
I don't remember the difference. This is something that was useful for debugging/testing for Sim at one point but I haven't had a need to use this in years.